docs/exploring-es-data.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[exploring-es-data]]
+= Exploring data in Elasticsearch
+
+[partintro]
+--
+By default Elastic APM data are stored in different indices, 
+in the format of `apm-[version]-[type]-[date]`.
```

Comment:
I'll add a line here saying that type is one of error, transaction, etc.; or a link to the event types section